astropy/nddata/bitmask.py

```diff
@@ -19,8 +19,7 @@
 
 
 _ENABLE_BITFLAG_CACHING = True
-_MAX_UINT_TYPE = np.maximum_sctype(np.uint)
-_SUPPORTED_FLAGS = int(np.bitwise_not(0, dtype=_MAX_UINT_TYPE, casting="unsafe"))
+_SUPPORTED_FLAGS = int(np.bitwise_not(0, dtype="uint64", casting="unsafe"))
```

Comment:
But what about 32-bit machine?

Reply:
It's been discussed already upstream: https://github.com/numpy/numpy/pull/24154#discussion_r1261538812
My understanding is that `uint64` is available *even on 32-bit* architectures, and is de facto always the result of `np.maximum_sctype(np.uint)`